doc/architecture/20170816-audit-plan-to-model-audit-data.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Term definitions
+
+A **theme** is a high level grouping criteria for all the content that GOV.Uk is currently hosting.
+
+An **inventory plan** is a set of rules that define the content that belongs to a theme. 
+
+The **inventory sheet** refines an inventory plan by excluding formats by adding/removing sets of content items.
```

Comment:
Inventory plans are not assigned to a Theme. They just model themes. Content Items are not moved between themes unless the rules are changed.